src/http/server.ts

```diff
@@ -315,17 +349,19 @@ export class Server {
    *     See `Drash.Http.Response.send()`.
    */
   public async handleHttpRequestError(
-    request: any,
-    error: any,
-    resource: Drash.Http.Resource | any = {},
-    response: Drash.Http.Response | any = {},
-  ): Promise<any> {
+    request: Drash.Http.Request,
+    error: Drash.Exceptions.HttpException,
+    resource: Drash.Http.Resource | null = null,
+    response: Drash.Http.Response | null = null,
+  ): Promise<Drash.Interfaces.ResponseOutput> {
```

Comment:
Has the doc block. been updated to reflect this change?

Reply:
updated